test/unit/lib/much-rails/call_method_callbacks_tests.rb

```diff
@@ -0,0 +1,187 @@
+require "assert"
+
+require "much-rails/call_method_callbacks"
```

Comment:
I may have told you wrong earlier, but the convention for requires in tests is:

```ruby
require "assert"
require "file_for_unit"

require "other_test_dependency1"
require "other_test_dependency2"

# ...
```
So can you remove the empty lines between `require "assert"` and the test unit file require? And add a empty line between those and any other test dependency requires please? 🙏🏻